modules/logging-loki-alerts.adoc

```diff
@@ -33,24 +33,79 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
---
-
-3. Apply the CR.
-
---
-include::snippets/logging-create-apply-cr-snip.adoc[lines=14..17]
---
+. Create an `AlertingRule` custom resource (CR):
++
+.Example infrastructure AlertingRule CR
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: AlertingRule
+  metadata:
+    name: loki-operator-alerts
+    namespace: openshift-operators-redhat <1>
+    labels: <2>
+      openshift.io/cluster-monitoring: "true"
+  spec:
+    tenantID: "infrastructure" <3>
+    groups:
+      - name: LokiOperatorHighReconciliationError
+        rules:
+          - alert: HighPercentageError
+            expr: | <4>
+              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"} |= "error" [1m])) by (job)
+                /
+              sum(rate({kubernetes_namespace_name="openshift-operators-redhat", kubernetes_pod_name=~"loki-operator-controller-manager.*"}[1m])) by (job)
+                > 0.01
+            for: 10s
+            labels:
+              severity: critical <5>
+            annotations:
+              summary: High Loki Operator Reconciliation Errors <6>
+              description: High Loki Operator Reconciliation Errors <7>
+----
+<1> The namespace where this `AlertingRule` CR is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
+<2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
+<3> `AlertingRule` CRs for `infrastructure` tenants are only supported in the `openshift-\*`, `kube-\*`, or `default` namespaces.
+<4> Value for `kubernetes_namespace_name:` must match the value for `metadata.namespace`.
+<5> Mandatory field. Must be `critical`, `warning`, or `info`.
+<6> Mandatory field.
+<7> Mandatory field.
++
+.Example application AlertingRule CR
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: AlertingRule
+  metadata:
+    name: app-user-workload
+    namespace: app-ns <1>
+    labels: <2>
+      openshift.io/cluster-monitoring: "true"
+  spec:
+    tenantID: "application"
+    groups:
+      - name: AppUserWorkloadHighError
+        rules:
+          - alert:
+            expr: | <3>
+            sum(rate({kubernetes_namespace_name="app-ns", kubernetes_pod_name=~"podName.*"} |= "error" [1m])) by (job)
+            for: 10s
+            labels:
+              severity: critical <4>
+            annotations:
+              summary:  <5>
+              description:  <6>
+----
+<1> The namespace where this `AlertingRule` CR is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
+<2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
+<3> Value for `kubernetes_namespace_name:` must match the value for `metadata.namespace`.
+<4> Mandatory field. Must be `critical`, `warning`, or `info`.
+<5> Mandatory field. Summary of the rule.
+<6> Mandatory field. Detailed description of the rule.
+
```

Comment:
Same comment as above regarding all of the list items needing to be grammatically parallel.